server/planning/feed_parsers/onclusive.py

```diff
@@ -159,6 +159,7 @@ def parse_location(self, event, item):
                     "name": event["venue"],
                     "qcode": "onclusive-venue:{}".format(venue_data.get("venueId")),
                     "address": self.parse_address(event),
+                    "location": {"lat": venue_data.get("locationLat"), "lon": venue_data.get("locationLon")},
```

Comment:
only populate `location` this if there is some data, afaik elastic would fail to save the item if there is `None`